### PR TITLE
add justMyCode debug option

### PR DESCRIPTION
**Pull request by yehiyam**
Add the `justMyCode` option to the debug configuration when debugging [.NET](https://github.com/OmniSharp/omnisharp-vscode/blob/master/debugger-launchjson.md#just-my-code) and [python](https://code.visualstudio.com/docs/python/debugging#_justmycode) with `debug (attach)`
The `justMyCode` property instructs the debugger to *not* skip loading external code.

### CODE REVIEW

**Comment by yehiyam** [comment]
@itowlson, @lstocchi, @bnookala 
Could someone take a look at the PR?

File: package.json [hunk 1]
```diff
@@ -1,1375 +1,1385 @@
 {
-    "name": "vscode-kubernetes-tools",
```
Comment (itowlson) [inline package.json:2]: Is the only change to this the whitespace?  GitHub doesn't make it clear what's happened but I wonder if it accidentally got changed from spaces to tabs?  It should be spaces with an indent of 4.

File: package.json [hunk 2]
```diff
@@ -1,1375 +1,1385 @@
...
+		"typescript": "^3.5.2",
+		"vscode-test": "^1.4.1",
+		"webpack": "^4.35.2",
+		"webpack-cli": "^3.3.5"
+	},
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/vscode-kubernetes-tools/vscode-kubernetes-tools"
+	},
+	"__metadata": {
```
Comment (itowlson) [inline package.json:1379]: Not sure where this came from - is it intentional?

**Review by itowlson — CHANGES_REQUESTED** [review]
Thanks for this!  It looks good to me except for the broad changes to `package.json` (where I think you meant only to add the new config setting) and `package-lock-json` (which I think doesn't need to change, though I could of course be wrong!).  Happy to merge once those are tidied up.

File: package.json (hunk 2 above)
Comment (yehiyam) [reply package.json:1379]: Not sure either. It wasn't intentional

File: package.json (hunk 1 above)
Comment (yehiyam) [reply package.json]: The indent was changed to tabs. Reverted back to spaces

**Review by itowlson — APPROVED** [review]
Looks good - thanks!

**Review by lstocchi — APPROVED** [review]

**Comment by itowlson** [comment]
@lstocchi thanks - I think this knocks off a reasonable round of PRs - shall we do a release or is there anything else you'd like to get in?

**Comment by lstocchi** [comment]
@itowlson +1 for a new release

**Pull request merged**